test/performance/odp_sched_perf.c

```diff
@@ -550,10 +593,17 @@ static int create_pool(test_global_t *global)
 		max_num = pool_capa.buf.max_num;
 		max_size = pool_capa.buf.max_size;
 		max_uarea = pool_capa.buf.max_uarea_size;
-	} else {
+		max_pools = pool_capa.event_vector.max_pools;
+	} else if (test_options->pool_type == ODP_POOL_PACKET) {
 		max_num = pool_capa.pkt.max_num;
 		max_size = pool_capa.pkt.max_seg_len;
 		max_uarea = pool_capa.pkt.max_uarea_size;
+		max_pools = pool_capa.event_vector.max_pools;
+	} else {
+		max_num = pool_capa.event_vector.max_num;
+		max_size = pool_capa.event_vector.max_size;
```

Comment:
Better to use some other variable than `max_size` here as max vector size is completely different compared to max buffer/packet size.